packages/dso-toolkit/src/components/tabs/tabs.scss

```
@@ -99,8 +99,9 @@
       display: block;
       line-height: typography.$line-height-base;
       margin-inline-end: 2px;
-      min-block-size: css-tabs-variables.$height;
-      padding: css-tabs-variables.$horizontal-padding css-tabs-variables.$vertical-padding;
+      min-block-size: css-tabs-variables.$block-size;
+      padding-block: css-tabs-variables.$inline-padding;
```

Dit lijkt me onwenselijk. `padding-block` vullen met een variabele `$inline-padding` en `padding-inline` vullen met een variabele `$block-padding`. Dan stel ik voor dat de waardes van beide variabelen worden omgedraaid en de juiste variabele op de juiste css-property wordt gezet. 

```suggestion
      padding-block: css-tabs-variables.$block-padding;
```

---

In dit geval gaan we hier wel mee wegkomen denk ik, want die vars worden beide maar 1 keer gebruikt zie ik. dan kunnen we het omdraaien, wel zo netjes. On it.

---

done